eng/common/scripts/Queue-Pipeline.ps1

```diff
@@ -58,10 +58,13 @@ param(
   [string]$VsoQueuedPipelines,
 
   # Already base 64 encoded authentication token
-  [string]$Base64EncodedAuthToken,
+  [string]$Base64EncodedAuthToken=$null,
```

Comment:
I suspect we don't have any more places we use the Base64EncodedAuthToken parameter anymore and if so let's just remove it.

Reply:
> I suspect we don't have any more places we use the Base64EncodedAuthToken parameter anymore and if so let's just remove it.
We cannot remove this at part of this PR, the script needs to handle both for a little while longer. I'd planned on updating [sync-directory.yml](sync-directory.yml) separately, after this PR has been merged and the [docindex pipelines will still be using the PAT until it can be replaced with the service principal](https://github.com/search?q=owner%3AAzure+azuresdk-apidrop-devops-queue-build-pat&type=code). The PATs need to be fully gone before this can be removed.

Reply:
Removed with commit 7.